tests/templates/templates_test.go

```diff
@@ -27,6 +27,9 @@ import (
 )
 
 func TestTemplates(t *testing.T) {
+	// Don't run in parallel.  We run so many templates we can easily go over our cloud limits
+	// t.Parallel();
```

Comment:
What limits do we hit?  Feels like it would be worth allowing these to run more in parallel - as I assume these are pretty expensive?

Reply:
I'll try to collect more information :)